src/BenchmarkDotNet/Extensions/ProcessExtensions.cs

```diff
@@ -128,6 +131,13 @@ internal static void SetEnvironmentVariables(this ProcessStartInfo start, Benchm
             if (benchmarkCase.Job.Environment.Runtime is MonoRuntime monoRuntime && !string.IsNullOrEmpty(monoRuntime.MonoBclPath))
                 start.EnvironmentVariables["MONO_PATH"] = monoRuntime.MonoBclPath;
 
+            if (benchmarkCase.Config.GetDiagnosers().OfType<PerfCollectProfiler>().Any())
+            {
+                start.EnvironmentVariables["COMPlus_PerfMapEnabled"] = "1";
+                start.EnvironmentVariables["COMPlus_EnableEventLog"] = "1";
+                start.EnvironmentVariables["COMPlus_EventSourceFilter"] = EngineEventSource.SourceName;
```

Comment:
@brianrob if I want to enable all events for given event source, is setting `COMPlus_EventSourceFilter` enough? 

The definition of event source that I care about:

https://github.com/dotnet/BenchmarkDotNet/blob/21a29406406aeca1cdfa195fa86ac28d04ba8e33/src/BenchmarkDotNet/Engines/EngineEventSource.cs#L7-L10

Reply:
Yes, that and setting `COMPlus_EnableEventLog=1`.  If you set `COMPlus_EventSourceFilter` to the EventSource that you want to capture, this will disable all other EventSources from being sent through to LTTng to avoid the cost of emitting all other EventSource events that may not be needed.